packages/terra-date-picker/src/DateInput.jsx

```diff
@@ -99,14 +98,21 @@ const defaultProps = {
   value: undefined,
 };
 
-// eslint-disable-next-line react/prefer-stateless-function
 class DatePickerInput extends React.Component {
   constructor(props) {
     super(props);
 
+    this.state = {};
```

Comment:
Can we set all the state values used in the component to undefined so we know what to expected face-up?